leveldb/key_test.go

```diff
@@ -16,7 +16,11 @@ import (
 var defaultIComparer = &iComparer{comparer.DefaultComparer}
 
 func ikey(key string, seq uint64, kt keyType) internalKey {
-	return makeInternalKey(nil, []byte(key), seq, kt)
+	ik, err := makeInternalKey(nil, []byte(key), seq, kt)
+	if err != nil {
+		panic(err) // Test helper - should not fail with valid parameters
+	}
+	return ik
 }
```

Comment:
I think this is fine -- it's a test, so not a big deal either way -- but for helper functions in tests like this, I like this pattern:

```golang
func ikey(t *testing.T, key string, seq uint64, kt keyType) internalKey {
	ik, err := makeInternalKey(nil, []byte(key), seq, kt)
	require.NoError(t, err, "making internal key")
	return ik
}
```

If we pass in `t *testing.T` then we can use test assertions to halt the test if we get an error, instead of having to rely on panics.

If this function is used widely in test code and you'd have to update a ton of places, I think it's fine to leave this as-is, too!